src/test/java/uk/gov/justice/hmpps/casenotes/controllers/NoteUsageByPrisonCodeIntTest.kt

```diff
@@ -0,0 +1,205 @@
+package uk.gov.justice.hmpps.casenotes.controllers
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments.of
+import org.junit.jupiter.params.provider.MethodSource
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.HttpStatus
+import uk.gov.justice.hmpps.casenotes.config.SecurityUserContext.Companion.ROLE_CASE_NOTES_READ
+import uk.gov.justice.hmpps.casenotes.legacy.dto.ErrorResponse
+import uk.gov.justice.hmpps.casenotes.notes.LatestNote
+import uk.gov.justice.hmpps.casenotes.notes.NoteUsageResponse
+import uk.gov.justice.hmpps.casenotes.notes.TypeSubTypeRequest
+import uk.gov.justice.hmpps.casenotes.notes.UsageByPrisonCodeRequest
+import uk.gov.justice.hmpps.casenotes.notes.UsageByPrisonCodeResponse
+import java.time.LocalDateTime
+
+class NoteUsageByPrisonCodeIntTest : IntegrationTest() {
+
+  @Test
+  fun `401 unauthorised`() {
+    webTestClient.post().uri(USAGE_URL).exchange().expectStatus().isUnauthorized
+  }
+
+  @Test
+  fun `403 forbidden - does not have the right role`() {
+    getUsageByPrisonCodeSpec(
+      request = UsageByPrisonCodeRequest(
+        typeSubTypes = setOf(TypeSubTypeRequest("T1")),
+        prisonCodes = setOf("NEO"),
+      ),
+      roles = listOf("ANY_OTHER_ROLE"),
+    ).expectStatus().isForbidden
+  }
+
+  @ParameterizedTest
+  @MethodSource("invalidPiUsageRequest")
```

Comment:
Incredibly minor. This is probably a copy paste name from the person identifier int tests